vm-virtio/src/console.rs

```diff
@@ -380,19 +376,24 @@ impl Console {
             acked_features: AtomicU64::new(0),
         });
 
+        let console_backend = VirtioBackend {
+            kill_evt: None,
+            avail_features,
+            acked_features: 0u64,
+            config_space: Vec::new(),
```

Comment:
You need to properly use the field `config_space` so that we can remove `config` from `Console`.

Reply:
Fixed.